app/api_mocks/case_service_mock.py

```diff
@@ -0,0 +1,9 @@
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
This needs to use structlogs wrap_logger. Or delete the log import as logger is not used.